library/default-synths-extra.scd

```diff
@@ -220,6 +225,7 @@
 		sound = MoogFF.ar(sound, SinOsc.ar(basefreq/32*rate, 0).range(lfof1,lfof2), resonance*4);
 		sound = MoogFF.ar(sound, min(env2*lfof2*1.1, 22000), 3);
 		sound = sound.tanh*5;
+		sound = sound * StrudelUtils.synthGain;
```

Comment:
No need to hardcode the volume adjustments, you can just do that by setting the `~amp` parameter in the input. So all the below can easily be avoided.